libs/wire-api/src/Wire/API/Conversation/Protocol.hs

```diff
@@ -109,7 +109,7 @@ optionalActiveMLSConversationDataSchema (Just v)
             schema
         <*> fmap (.epochTimestamp)
           .= field "epoch_timestamp" (named "EpochTimestamp" . nullable . unnamed $ utcTimeSchema)
-        <*> maybe MLS_128_DHKEMX25519_AES128GCM_SHA256_Ed25519 (.ciphersuite)
+        <*> maybe MLS_128_DHKEMP256_AES128GCM_SHA256_P256 (.ciphersuite)
```

Comment:
This looks like a mistake, we shouldn't change legacy defaults in older versions.